src/Controls/StreetControls.js

```diff
@@ -397,6 +399,14 @@ class StreetControls extends FirstPersonControls {
             this.view.notifyChange(this.view.camera.camera3D);
         }
     }
+
+    dispose() {
+        this.view.domElement.removeEventListener('mouseout', this._onMouseOut, false);
+
+        super.dispose();
+
+        this.dispatchEvent({ type: 'dispose' });
```

Comment:
could remove `this.dispatchEvent`, it's useless because is already called in `super.dispose();`